keps/sig-node/0035-20190130-topology-manager.md

```diff
@@ -195,14 +199,22 @@ The Topology Hint currently consists of
       * For each Hint Provider, there is a possible resource assignment that satisfies the request, such that the least possible number of NUMA nodes is involved (caculated as if the node were empty.)
       * There is a possible assignment where the union of involved NUMA nodes for all such resource is no larger than the width required for any single resource.
 
+#### Scopes
+
+The scope determines the basis of how topology manager collects topology hints, this also affects how policy works(container basis or pod basis).
+
+**container (default)**: Topology Manager collects topology hints from the Hint Providers per container basis, and a policy provides an allocation per container basis. A Pod will be admitted if the provided container basis allocation meets the criteria of configured policy.
+
+**pod**: Topology Manager collects topology hints per pod basis, and a policy provides an allocation per pod basis. A Pod will be admitted if the provided pod basis allocation meets the criteria of configured policy.
+
```

Comment:
The wording here makes this a little confusing.  Try changing it to the following:

> The Topology Manager will attempt to align resources on either a pod-by-pod or a container-by-container basis depending on the value of a new kubelet flag, `--topology-manager-scope`.  The values this flag can take on are detailed below:
>
>\*\*container (default)\*\*: The Topology Manager will collect topology hints from from all Hint Providers on a container-by-container basis.  Individual policies will then attempt to align resources for each container individually, and pod admission will be based on whether all containers were able to achieve their individual alignments or not.
>
> \*\*pod\*\*: The Topology Manager will collect topology hints from from all Hint Providers on a pod-by-pod basis.  Individual policies will then attempt to align resources for all containers collectively, and pod admission will be based on whether all containers are able to achieve a common alignment or not.